TemplateApplication/Supporting Files/TemplateApplication.docc/TemplateApplication.md

```diff
@@ -18,73 +18,73 @@ The Spezi Template Application demonstrates using the [Spezi](https://github.com
 
 > Tip: Do you want to try out the Spezi Template Application? You can download it to your iOS device using [TestFlight](https://testflight.apple.com/join/ipEezBY1)!
 
-The following screenshots show a wide variety of features based on Spezi Modules that are part of the Spezi Template Application.
+The following screenshots show a wide variety of features based on Spezi Modules that are part of the Spezi Template Application. Click on the links below the screenshots to learn more about the specific Spezi module.
 
 @Row(numberOfColumns: 3) {
     @Column(size: 1) {
         @Image(source: "Welcome", alt: "A screen displaying welcome information.") {
-            Welcome View.
+            [Welcome View](https://spezi.health/SpeziTemplateApplication/documentation/templateapplication/welcome)
```

Comment:
Maybe we can use direct DocC links here that would point right to the documentation in Xcode. We might want to test this by building the documentation in Xcode and seeing that all links work without the need to visit the SPI website. Applies to all other changes as well.

E.g. this should work as a link:
```swift
``Welcome``
```